main/src/cgeo/geocaching/settings/fragments/PreferenceMapFragment.java

```diff
@@ -56,6 +56,7 @@ private void initMapSourcePreference() {
         }
         prefMapSources.setEntries(entries);
         prefMapSources.setEntryValues(values);
+        // todo
```

Comment:
Selecting combined as map source doesn't have a direct effect, related?

Reply:
Old code:
```
-        if (isPreference(preference, R.string.pref_mapsource)) {
-            // reset the cached map source
-            MapSource mapSource;
-            try {
-                mapSource = MapProviderFactory.getMapSource(stringValue);
-            } catch (final NumberFormatException e) {
-                Log.e("SettingsActivity.onPreferenceChange: bad source id '" + stringValue + "'", e);
-                mapSource = null;
-            }
-            // If there is no corresponding map source (because some map sources were
-            // removed from the device since) then use the first one available.
-            if (mapSource == null) {
-                mapSource = MapProviderFactory.getAnyMapSource();
-                if (mapSource == null) {
-                    // There are no map source. There is little we can do here, except log an error and
-                    // return to avoid triggering a null pointer exception.
-                    Log.e("SettingsActivity.onPreferenceChange: no map source available");
-                    return true;
-                }
-            }
-            Settings.setMapSource(mapSource);
-            preference.setSummary(mapSource.getName());
-        }

Reply:
Also show the current selection as summary...